lib/intersection_validator.rb

```diff
@@ -0,0 +1,10 @@
+class IntersectionValidator < ActiveModel::EachValidator
```

Comment:
What do you think about put this class inside the scope of `ActiveModelValidatesIntersectionOf` module, to avoid conflicts when someone require this gem.
What do you think?